config/op-consumer/mapping/queries/op2dl/mapping/organization.sparql

```diff
@@ -0,0 +1,17 @@
+CONSTRUCT {
+  ?s a <http://www.w3.org/ns/org#Organization> . 
+}  
+WHERE {
+  ?s a <http://data.vlaanderen.be/ns/besluit#Bestuurseenheid> .
+
+  FILTER NOT EXISTS {
+    VALUES ?type {
+      <http://data.lblod.info/vocabularies/erediensten/BestuurVanDeEredienst>
+      <http://data.lblod.info/vocabularies/erediensten/CentraalBestuurVanDeEredienst>
+      <http://data.lblod.info/vocabularies/erediensten/RepresentatiefOrgaan>
+      <http://purl.org/dc/terms/Agent>
```

Comment:
Aren't they all Agents when coming from OP?

Reply:
Seems not cause the query was giving me results..? I didn't question too much the config, I took it as it was, just split it :grimacing: